cluster/manifests/role-sync-controller/cronjob.yaml

```diff
@@ -0,0 +1,21 @@
+{{ if .Cluster.ConfigItems.role_sync_controller_enabled "true" }}
```

Comment:
Since this is behind a feature flag, we should also have entries in `deletions.yaml` to clean it up if first enabled and then disabled again.

Reply:
ah, good point. I'll add.